src/main/java/Duke.java

```diff
@@ -1,10 +1,165 @@
+
+    //CONSTANTS
+    public static final String SEPARATOR = "____________________________________________________________\n";
+
+    //METHODS
+    public static void printGreeting() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n"
+                + logo
+                + SEPARATOR
+                + " Hello! I'm Duke\n"
+                + " What can I do for you?\n"
+                + SEPARATOR);
+    }
+
+    public static void printError() {
+        System.out.println(SEPARATOR
+                + " Oops, something went wrong!\n"
+                + SEPARATOR);
+    }
+
+    public static String[] organize(String userInput) {
+        String[] organizedUserInput = new String[3];
+        if (!userInput.contains(" ")) {
+            //command is list
+            organizedUserInput[0] = userInput;
+        } else {
+            int spaceIndex = userInput.indexOf(" ");
+            String command = userInput.substring(0, spaceIndex);
+            organizedUserInput[0] = command;
+            if (!userInput.contains("/")) {
+                //command is either done or todo
+                String detail = userInput.substring(spaceIndex + 1);
+                organizedUserInput[1] = detail;
+            } else {
+                //command is either deadline or event
+                int slashIndex = userInput.indexOf("/");
+                String detail = userInput.substring(spaceIndex + 1, slashIndex - 1);
+                String time = userInput.substring(slashIndex + 1);
+                organizedUserInput[1] = detail;
+                organizedUserInput[2] = time;
+            }
+        }
+        return organizedUserInput;
+    }
+
+    public static void printTaskList(Task[] taskList, int taskListSize) {
+        System.out.println(SEPARATOR
+                + " Here are the tasks in your list:");
+        for (int i = 0; i < taskListSize; i++) {
+            int j = i + 1;
+            System.out.println(" " + j + "." + taskList[i]);
+        }
+        System.out.println(SEPARATOR);
+    }
+
+    public static void markAsDone(Task[] taskList, int taskListSize, int taskNumber) {
+        if (taskNumber > taskListSize - 1) {
+            printError();
+        } else {
+            taskList[taskNumber].setDone();
+            System.out.println(SEPARATOR
+                    + " Nice! I've marked this task as done:\n"
+                    + "  " + taskList[taskNumber] + "\n"
+                    + SEPARATOR);
+        }
+    }
+
+    public static Task createTask(String[] organizedInput) {
+        String category = organizedInput[0];
+        String description = organizedInput[1];
+        String details = organizedInput[2];
```

Comment:
Good that you defined variables to hold the different values, instead of just referring to the array! Made your code very clear.